ui/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN rpm -i https://download.oracle.com/otn_software/linux/instantclient/1912000/
     ldconfig
 RUN python3 -m pip install --no-cache-dir --upgrade pip
 RUN python3 -m pip install --no-cache-dir --upgrade setuptools
-RUN rm -rf /usr/lib/python3.6/site-packages/ipaddress*
```

Comment:
I think this line can be completely removed now.

Reply:
Even better. Done.